stdlib/LinearAlgebra/src/eigen.jl

```diff
@@ -46,15 +46,37 @@ julia> vals, vecs = F; # destructuring via iteration
 julia> vals == F.values && vecs == F.vectors
 true
 ```
+
+Depending on the constructing algorithm, an object `F::Eigen` may optionally also contain:
+
+* the left eigenvectors as the columns of the matrix `F.vectorsl`,
+* the reciprocal condition numbers of the eigenvalues in the vector `F.rconde`,
+* the reciprocal condition numbers of the eigenvectors in the vector `F.rcondv`,
+* the norm (usu. `p=1`) of the balanced form of the matrix in `F.balnorm`.
 """
-struct Eigen{T,V,S<:AbstractMatrix,U<:AbstractVector} <: Factorization{T}
+struct Eigen{T,V,S<:AbstractMatrix,U<:AbstractVector,R<:AbstractVector,B} <: Factorization{T}
     values::U
     vectors::S
-    Eigen{T,V,S,U}(values::AbstractVector{V}, vectors::AbstractMatrix{T}) where {T,V,S,U} =
-        new(values, vectors)
+    vectorsl::S
+    unitary::Bool # indicates that the vectors form a unitary matrix (normality)
```

Comment:
This could be understood as the matrix itself being unitary, why not just "normal"?

Reply:
I'm following Klaus here, who emphasized unitarity as a property of the decomposition itself. I don't have a strong opinion either way, and will switch to "normal" if no one else objects.